README.md

```diff
@@ -8,11 +8,47 @@ Supports both V1 (Provision) and V3 of anisette servers.
 
 ## Setup
 
-First, [download the Apple Music APK](https://apps.mzstatic.com/content/android-apple-music-apk/applemusic.apk) and extract it as a zip. (`cd tmp && unzip ../applemusic.apk`) Then, move the `lib`
-directory to the directory you want to run omnisette-server from.
+First, [download the Apple Music APK](https://apps.mzstatic.com/content/android-apple-music-apk/applemusic.apk) in the root of the project directory and then extract it as a zip.
 
-Next, [download the latest omnisette-server binary for your platform from GitHub Releases](https://github.com/SideStore/omnisette-server/releases). Place it in the same directory with the `lib`
+```bash
+unzip applemusic.apk 'lib/*/libstoreservicescore.so' 'lib/*/libCoreADI.so'
+```
+
+Next, [download the latest omnisette-server binary for your platform from GitHub Releases](https://github.com/SideStore/omnisette-server/releases). Place it in the root project directory with the `lib`
```

Comment:
Wait, if they're downloading the precompiled binary why do they need the repo cloned?

Reply:
Ah yeah, I misread that somehow...